sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -83,6 +83,13 @@ public DynamoDBContextConfig()
         /// </summary>
         public string TableNamePrefix { get; set; }
 
+        /// <summary>
+        /// The object persistence model API relies on an internal cache of the DynamoDB table's metadata to construct and validate 
+        /// requests. This controls how the cache key is derived, which influences when the SDK will call DescribeTable
```

Comment:
nit: could you add a doc reference to "DescribeTable"

Reply:
Addressed in https://github.com/aws/aws-sdk-net/compare/8a4f5a780d437debec1f38696b9dc2f069b991c1..1161dc30f02a5a18108c28b0191f21da5080a368